src/containers/Home/BlockList/index.tsx

```diff
@@ -1,24 +1,76 @@
+import { useEffect, useState } from 'react'
 import Button from '@/components/Button'
 import Date from '@/components/Date'
 import { AddIcon } from '@/components/Icons'
 import Block from '@/components/Block'
 import NoData from '@/components/NoData'
-import { MOCK_BLOCK_LIST } from '@/constants/block'
 import useSelectedDateState from '@/store/selectedDate'
+import { dayBlockAPI } from '@/api'
+import { GetDayBlocksResponse } from '@/api/types/base.types'
+import { BASE_URL } from '@/constants/urls'
 import DiaryButton from './DiaryButton'
 
+import rnWebViewBridge from '@/utils/react-native-webview-bridge/new-webview/rnWebViewBridge'
+
+const AddButton = ({
+  text,
+  onClick,
+}: {
+  text: string
+  onClick: () => void
+}) => {
+  return (
+    <Button
+      color="gray"
+      fontWeight="bold"
+      className="py-[11px]"
+      onClick={onClick}
+    >
+      <AddIcon className="!fill-textGray-200" width={18} height={18} />
+      {text}
+    </Button>
+  )
+}
+
 const BlockList = () => {
-  const selectedDate = useSelectedDateState((state) => state.date) // TODO: 해당 날짜의 api 불러올 것
-  const { date, totalBlock, totalTask, blocks } = MOCK_BLOCK_LIST
+  const selectedDate = useSelectedDateState((state) => state.date)
+  const [data, setData] = useState<GetDayBlocksResponse | null>(null)
 
-  const handleBlockCreate = () => {
-    // 블럭 생성 페이지로 이동
+  const handleAddSavedBlock = () => {
+    rnWebViewBridge.open({
+      key: 'savedBlock',
+      url: `${BASE_URL}/blocks/saved`,
```

Comment:
```suggestion
      url: `${BASE_URL}/blocks/saved?date=${selectedDate}`,
```

블럭을 불러 올 날짜를 블럭 불러오기 페이지에서 알아야 할 것 같습니다.

Reply:
블럭 추가하려면 날짜를 알아야겠네요, 추가했습니다 감사합니다:) [13209b3](https://github.com/dnd-side-project/dnd-8th-7-frontend/pull/52/commits/13209b3bc6854c6cc7bc28d16f89d7d128c907a1)